docs/install_guides/installing-openvino-from-archive-linux.md

```diff
@@ -125,7 +125,16 @@ See the [Release Notes](https://www.intel.com/content/www/us/en/developer/articl
       cd /opt/intel/openvino_2022.3.1
       sudo -E ./install_dependencies/install_openvino_dependencies.sh
 
-6. For simplicity, it is useful to create a symbolic link as below:
+6. Install *numpy* Python Library:
+
+   You can use the ``requirements.txt`` file from the ``opt/intel/openvino_2022.3.1/python/python.<x>`` folder:
+
+   .. code-block:: sh
+
+      cd /opt/intel/openvino_2022.3.1
+      pip3 install -r ./python/python3.<x>/requirements.txt
```

Comment:
I propose:
- Make this step with numpy installation to be optional (maybe last one), because not archive users want to use python. Some users can be satisfied with C++ API only.
- Let's have numpy installation steps only for Archives installation type, because other types don't have python in 2022.3 2023.0 releases (except pip, which automatically installs numpy as openvino depenendecy). In 2023.1 conda-forge, brew, apt will contain python content and also automatically install numpy. So, only archives require explicit (nut optional) step with numpy installation.

Reply:
I have removed this step from all except archive installation methods with the https://github.com/openvinotoolkit/openvino/pull/19093/commits/24c687da441b4c62e75c44e64e129a73cb599b1c commit.

Reply:
looks better, thanks!
do you think it's useful to mark numpy installation step as (Optional) and note, that it's for cases when you want to use Python API?